src/bake/hmr-module.ts

```diff
@@ -940,3 +897,13 @@ if (side === "client") {
     onServerSideReload: cb => (onServerSideReload = cb),
   });
 }
+
+// The following API may be altered at any point.
+// Thankfully, you can just call `import.meta.hot.on`
+let testingHook = globalThis['bun do not use this outside of internal testing or else i\'ll cry'];
+testingHook && testingHook({
```

Comment:
```suggestion
testingHook?.({
```

Reply:
today i learned this wont eval the args if the fn is undefined. cozy